flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/checkpointing/AlternatingControllerTest.java

```diff
@@ -259,6 +298,47 @@ private void testTimeoutBarrierOnTwoChannels(
         assertData(gate);
     }
 
+    //    @Test
+    //    public void testActiveTimeoutAlignment() throws Exception {
+    //        int numberOfChannels = 2;
+    //        SingleInputGate inputGate =
```

Comment:
I guess this is some left-over that should be removed in this commit, not the next one?

Could also please make the commit message more informative?
(`[refactor] Extracted builders`)